### DISCUSSION TITLE
Welcome to the Firefly Discussion Forum!

### DISCUSSION THREAD

**Discussion by cvarley100** [iotaledger/firefly#451] [Announcements]
## 👋 Welcome!
  The Firefly Discussion Forum is a place to connect with other members in our community. __We encourage everyone to__...
  * __be curious and ask questions that you have__.
  * __share and discuss ideas about Firefly__.
  * __welcome others to our community and be open-minded (remember that this is a community we build together :muscle:)__.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Phyloiota** [comment]
Hey - i just opened an issue as i was able to import a stronghold file to create a new profile, while the profile that was represented by that stronghold file was still an active and in use profile in firefly. 
I was wondering it it is a bug or a feature....
My understanding is... The stronghold file backed up during profile creation contains your encrypted seed. 
So if i am able to use this file (seed) for a new profile i would actually have 2 profiles accessing funds from the same seed. Could this lead to double spends or lost funds or confusion...? 
The wallet did not allow to generate addresses (create account) with the already used stronghold, but still i was able to create a valid profile. I think thats already to far in the setup process...   ... if its really a bug...

On the other hand i thought ... if there will be a mobile version and i only have one profile (seed) and i want to access funds from mobile now after i created the account in desktop i would anyway have to enter the seed (upload the stronghold file from my existing account to the mobile version.... so it should be possible....

Hope for some clarification on that...

**Comment by admljhnsn** [comment]
it'd be nice if the price chart showed the current price like "$5.00" instead of just a line

**Comment by Iota2021** [comment]


**Reply by admljhnsn** [reply]
the migration hasn't started yet. tomorrow (April 21) is the first day. will be posts about it on blog.iota.org i imagine

**Comment by reddevil1212** [comment]
Hi together,

where can i check the file hashes of the downloaded files of your website? 
Would be good if you post them somewhere were it can be easily seen
thx

**Reply by Thoralf-M** [reply]
The file hashes aren't on the website, but here https://github.com/iotaledger/firefly/releases
And here is a guide which explains how to verify it https://chrysalis.docs.iota.org/firefly/verify_download.html

**Comment by Travetown** [comment]
Unfortunately, despite several questions (yes, also in Discord), I have not yet received an answer.  So again:
 Will there be a stronghold file for ledger accounts in Firefly, in which the transaction history is stored, or does it slowly disappear with pruning, or did it disappear after a restore?

**Comment by jayscee** [comment]
I started migration from Trinity to Firefly on 28th and message came up saying to close firefly website due to upgrade, so migration was aborted.  So I now have an empty firefly wallet and tokens are still in Trinity.  On firefly.iota.org website it states "If any part of the process is aborted or fails, it can be redone!", but no indication as to how to redo it.  Does anyone know how to redo it?

**Comment by rowacle** [comment]
Hi, I did the migration proceses before apr 28 all ok, but today I opened the wallet and the funds are still bloqued, some one can tell me what I have to do to have the funds available in the balance? Thanks

**Comment by benj2366** [comment]
I am benj2366, i have wallet in firefly with a profile name of benj2366 i am currently staking ASMB and SMR, however right now i cannot open my firefly wallet, when i input my 6 digit PIN nothing happens its just like it stuck,, I do have another firefly wallet in my Laptop and it is working fine, this one is in my Desktop and it has the same profile name of benj2366, can anyone help me,,,???

**Comment by benj2366** [comment]
Firefly v1.3.3
Language: en
Currency: USD
Node selection: Automatic
Platform: win32
Platform Version: 6.3.9600
Platform Architecture: x64
CPU Count: 4
Total Memory: 16266.5 MB
Free Memory: 13642.0 MB

**Comment by benj2366** [comment]
I am afraid my fund in that wallet will be lost

**Comment by 1979aea2022** [comment]
Hello.
I am missing many Shimmers in the claim, a wallet with a very small balance that appears as claimed was restored, the problem was the one that was giving search for rewards and the balance was incomplete and I gave it to claim reward by mistake and at the time the claim how long it lasts I left the computer alone it turned off and when I restored the claim process a wallet appeared with nothing more than a very small claimed balance, and the other two wallets did not appear that being at the beginning of the search process there were the three wallets, I'm going to look in the iota / explorer to see what appears, again thank you very much

**Reply by benj2366** [reply]
I do have three wallets which has small balance i want to consolidate it, is it possible

**Comment by vivirsiendo** [comment]
hola a todos quería pediros si podriais informarme pork despues de tener summer en firafly ya no los veo por ningún lado.Gracias

**Reply by begonaalvarezd** [reply]
shimmer no sale en las nuevas versiones de firefly porque están en su propia network, puedes seguir estos pasos https://blog.shimmer.network/how-to-claim-smr-firefly/

**Comment by bonnielmckenzie** [comment]
Hello to all.  Some years ago I stored some IOTA crypto on Firefly.  I still have my username & a 6 digit number for signing in.  Unfortunately, my husband was ill with cancer over the last few years & I simply lost all interest in crypto.  Now that he has passed away & the more urgent matters have been attended to, I am interested in recovering my account.  It looks like Firefly has undergone some changes that I don't understand & I am no longer able to simply sign in to my account as was previously easily possible.  Is there a IOTA help desk (as in a real person) for people like me?

**Reply by begonaalvarezd** [reply]
Hi, first of all im very sorry for your loss 🌷 
Firefly never had the concept of "username" but I undestand the misconception. Your wallet can be recovered at any time using the 24 words mnemonic that you were given when you first created the wallet, you can always use it to restore your funds using any compatible wallet like Firefly or Bloom. If you lost your 24 words you can try locating the "stronghold" file which is a backup, which needs a password to be decrypted (not the 6 pin digits, but a password) and you can use that backup to recover too. 

If you need further assistance, I suggest you to join our discord and post your ticket there, github is not really our support channel https://discord.iota.org/

Wish you luck 🙏🏼 and I hope you can recover your funds

**Reply by bonnielmckenzie** [reply]
Thank you for your kind reply.  It's been so long since I set up my Firefly wallet that I don't recall a 24 word mnemonic.  So, if there was such a thing, I don't seem to have it.  However, I do have what I thought was a username, but it may actually be the stronghold password.  When I originally set up my wallet, all I had to do was open up Firefly on my laptop & enter my PIN.  That was that simple, but no longer possible.  Again, appreciate your thoughtful reply.

**Reply by begonaalvarezd** [reply]
If your profile exists in Firefly you should still be able to enter to your existing profile using your 6 digit pin, why arent you able to do that anymore? Did you change the machine? If you dont have any existing profile in your Firefly wallet, you have to recover it using your 24 words mnemonic, or a stronghold file and its password.

**Reply by bonnielmckenzie** [reply]
I tried Discord & was told that it is not affiliated with Firefly Wallet & is not able to provide support for my issue.
I'm still using the same laptop, but I deleted Firefly from my laptop last summer when I was not able to get it to work fore me anymore.  It simply would no longer offer me the option to enter my 6 digit pin.  Maybe it was an outdated version or it 'forgot me' since I hadn't logged in for years.  I obviously have no idea what explains the problem I was having.  Anyway, I've downloaded the latest version &, under the recovery option, just for pure the heck of it, I entered my 24 word mnemonic for my Ledger wallet.  (Note, I have never any way linked Firefly with Ledger.)  This allowed me to set up a password & PIN.  I used the a different 'password' from my original Firefly Wallet, but did use my original PIN.  Once in the wallet, my balance shows as zero.  So, I think I actually just created a new wallet & did not access my original wallet, even though this was all done as a 'recovery' effort.  While this is a fun challenge, it's literally not worth all of this time & effort.

**Reply by begonaalvarezd** [reply]
I cant find your support ticket in our discord, are you sure you opened a ticket in our official discord at the "**raise-support-ticket**" channel? 

The pin has nothing to do with your keys, is just a security measure within Firefly to keep your profile locked. Firefly is a self custody wallet, where you are the owner of your own keys. If you originally depositted funds to an account using a mnemonic that you do not have anymore, there is nothing we can do, this is how self custody wallets are. If you have a ledger device and you used it originally to set up your firefly wallet you can install the IOTA app from ledger live and create a profile in Firefly using connecting it to your ledger device, as a recommendation you should not be using the 24 words of your ledger inside any software app, that defeats the whole purpose of having a ledger device.

If you dont have your original keys I am sorry but I dont think we can help you. You can try importing to firefly all your mnemonics and connecting all your ledger devices and run a balance finder from settings > advanced > balance finder, to see if any of the mnemonics have any funds in any address

Wish you luck 🌷

**Comment by bonnielmckenzie** [comment]
My Discord ticket number is 52835256, if that helps.  I reached Discord
through Firefly, so I presumed it was a legitimate site.  Are the “keys”
the same thing as the 24 word mnemonic?  I did not use my Ledger device to
set up my Firefly wallet.  If I “import to Firefly all of my mnemonics”,
 wouldn’t that go against the recommendation in your last email?  Thank you
for your time & effort to help me out, but again, I doubt if my balance in
Firefly is enough to fret over.  This is, however, a captivating challenge!
Bonnie
